pyquil/api/tests/test_base_connection.py

```diff
@@ -0,0 +1,336 @@
+import pytest
+import requests_mock
+import urllib.parse
+
+from pyquil.api._base_connection import ForestSession
+from pyquil.api._config import PyquilConfig
+from pyquil.api._errors import UserMessageError
+from pyquil.tests.utils import api_fixture_path
+
+
+@pytest.fixture
```

Comment:
a fixture is a special construct that is created as part of the test suite -- it looks like this is just a function that returns a dictionary -- if you just put this as a constant at the top of the file it would do the exact same thing no?

Reply:
if the fixture is parameterized, or if you are going to edit it in the tests and need a fresh instance each test, that is a reason to make a fixture -- but i think you're just treating it as a constant